src/Compilers/Test/Resources/Core/NetFX/Minimal/mincorlib.cs

```diff
@@ -2,8 +2,8 @@
 
 namespace System
 {
-	public class Object {}
-    public struct Byte {}
+    public class Object { }
```

Comment:
Instead of modifying mincorlib for everybody and making it less minimal, could we simply create library of the right shape for the test that need it (preferably in memory)?